src/transformers/modeling_tf_albert.py

```diff
@@ -478,9 +478,10 @@ def call(self, hidden_states):
         return hidden_states
 
 
+@keras_serializable
 class TFAlbertMainLayer(tf.keras.layers.Layer):
     def __init__(self, config, **kwargs):
-        super().__init__(config, **kwargs)
+        super().__init__(**kwargs)
```

Comment:
Bugfix, independent of this PR and posted separately in #3115, but included here because it got in the way of testing.